plugins/blockvolume/api/types.go

```diff
@@ -11,9 +11,23 @@ type BlockVolumeInfo struct {
 	HaCount int    `json:"hacount,omitempty"`
 }
 
+// HostVolumeInfo represents block volume info
+type HostVolumeInfo struct {
+	// HostVolReplicaCnt represents the replica count of the block hosting volume
+	HostVolReplicaCnt int `json:"hostvolreplicacnt,omitempty"`
+	// HostVolThinArbPath represents the thin arbiter path
+	HostVolThinArbPath string `json:"hostvolthinarbpath,omitempty"`
+	// HostVolShardSize represents the shard size of the block hosting volume
+	HostVolShardSize uint64 `json:"hostvolshardsize,omitempty"`
+	// Size represents Block Hosting Volume size in bytes
+	HostVolSize uint64 `json:"hostsize,omitempty"`
+}
+
 // BlockVolumeCreateRequest represents req Body for Block vol create req
 type BlockVolumeCreateRequest struct {
 	*BlockVolumeInfo
+	HostVolumeInfo
+	BlkType  string   `json:"blktype,omitempty"`
```

Comment:
what is `blktype`?

Reply:
It will contain the filesystem type the block will be formatted with, if its "none" then the file will not be formatted with any filesystem. The default is "xfs".

Reply:
can we make it as `block-type` for more readability and set a default value on server side based on the empty string will be a good close instead of "none"

Reply:
Sure, may be will use "raw" instead of "none"?